scss/share-component.scss

```diff
@@ -26,3 +26,10 @@
 		}
 	}
 }
+
+// Override default framework styles
+.field {
+	.chzn-container {
+		display: inline-block;
+	}
+}
```

Comment:
yeah what i'm saying is it might be better to remove the override that was added in https://github.com/silverstripe/silverstripe-framework/commit/730c4d37 rather than adding another override which overrides another override